reference/user_defined_functions.rst

```diff
@@ -719,6 +718,35 @@ Example
   --------------
   73 MB
 
+citus.find_groupid_for_node
```

Comment:
I don't think this is meant to be external. It also seems low-level. `master_add_secondary_node` probably should be added though. The function comment is here: https://github.com/citusdata/citus/blob/f1b51d7bbe4178e2cd5fb08959e7a4cd87bd6a3b/src/backend/distributed/utils/node_metadata.c#L174. The arguments are different from this and master_add_node. In this function, we also specify which is the primary node for this one.